src/components/Modals/FiatRamps/config.ts

```diff
@@ -115,4 +117,27 @@ export const supportedFiatRamps: SupportedFiatRamp = {
       }
     },
   },
+  MtPelerin: {
+    label: 'fiatRamps.mtPelerin',
+    tags: ['fiatRamps.noKYC', 'fiatRamps.nonUS'],
+    logo: MtPelerinLogo,
+    isImplemented: true,
+    supportsBuy: true,
+    supportsSell: true,
+    // https://developers.mtpelerin.com/service-information/pricing-and-limits#limits-2
+    // 50 CHS is currently equivalent to 51.72 USD, maybe get the actual rate from redux fiat market data?
```

Comment:
Also might be good to add an additional comment line explaining Mt Pelerin has a minimum of 50 CHF, and our fiat balance is denoted in USD